src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -355,6 +357,12 @@ class CommonVisitor : public AST::BaseVisitor<Derived> {
         return v;
     }
 
+    void handle_attribute(ASR::symbol_t *s, std::string attr_name,
+                const Location &loc, Vec<ASR::expr_t*> &args) {
+        tmp = attr_handler.get_attribute(s, attr_name, al, loc, args);
```

Comment:
To be precise, since `handle_attribute` is a method, no need to pass `diag` in it, but pass it in here into `get_attribute`.